exericse4/exercise3.js

```diff
@@ -0,0 +1,4 @@
+const sum = (array) => {
```

Comment:
Function name must be more descriptive, like sumOfElements, arraySum .. and many more